insitu/src/filtered_view.cpp

```diff
@@ -37,7 +37,7 @@ FilteredView::FilteredView(const ros::NodeHandle& parent_, QString _name,
 
     // graphics view for rendering filters
     filterScene = new QGraphicsScene(this);
-    filterScene->setBackgroundBrush(QBrush(Qt::lightGray));
+    filterScene->setBackgroundBrush(QBrush(Qt::black));
```

Comment:
I don't mind this change but a more permanent solution would probably be to expose this as a setting in a `Preferences` panel or something.

Reply:
Sounds good. Please let us know when you get round to adding this feature.